.github/contributing_templates/notebook/example_class.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class ExampleClass:
```

Comment:
I would suggest having more meat to this class, for example it could be a simple `nn.Module` class that squares its inputs in `forward` or something else super simple. This class should be imported in the notebook and used to do something really simple just to illustrate a regular pattern for doing this.

Reply:
Thanks for the suggestion @ericspod .

I don't want to introduce `nn.Module` in the python script because it will be more like an example of PyTorch instead of MONAI. But I agree with you that it should be more meat in the class. One option that may be a good fit is toy dataset generation with `create_test_image_2d` or `create_test_image_3d`.

Reply:
Here is what I put into the class:

```python
class ExampleImageGenerator():
    def __init__(self, num_image=40, image_size=(128, 128)):
        self.num_image = num_image
        self.image_size = image_size
    
    def generate(self, tempdir):
        for i in range(self.num_image):
            im, seg = create_test_image_2d(
                self.image_size[0], self.image_size[1], num_seg_classes=1,random_state=np.random.RandomState(42)
            )
            Image.fromarray((im * 255).astype("uint8")).save(os.path.join(tempdir, f"img{i:d}.png"))
            Image.fromarray((seg * 255).astype("uint8")).save(os.path.join(tempdir, f"seg{i:d}.png"))
        
        images = sorted(glob(os.path.join(tempdir, "img*.png")))
        segs = sorted(glob(os.path.join(tempdir, "seg*.png")))
        return (images, seg)
```